apps/game_backend/lib/game_backend/transaction.ex

```diff
@@ -0,0 +1,14 @@
+defmodule GameBackend.Transaction do
+  @moduledoc """
+  Module for exponsing transactions to game applications.
+  """
+
+  alias GameBackend.Repo
+
+  @doc """
+  Run a multi transaction.
+  """
+  def run(multi) do
+    Repo.transaction(multi)
+  end
```

Comment:
What is the motivation of doing this instead of just use the repo transaction in the multi?